google-bigquery/1.1/modules/ROOT/pages/google-bigquery-connector-reference.adoc

```diff
@@ -465,9 +465,7 @@ Connector does not stream the request content. |  |
 === Get Query Results
 `<bigquery:get-query-results>`
 
-
-Get results of a query. This operation makes an HTTP GET request to the /files/{fileId}/comments endpoint.
-
+This operation makes an HTTP GET request to the projects/{projectId}/queries/{jobId} endpoint.
```

Comment:
should we add the operation description back in?

Reply:
Nah, I think it's self-evident. It didn't add much.